app/views/users/new.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= form_for @user do |f| %>
+  <%= f.label :username %>
+  <%= f.text_field :username %>
+  <%= f.label :password %>
+  <%= f.password_field :password %>
+  <%= f.label :address %>
+  <%= f.text_field :address %>
```

Comment:
I think it's a good idea we included this here. For now it will make the process a bit smoother, but I think later we should look into only requiring the address at checkout.